public/index.html

```diff
@@ -4,8 +4,11 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Add todo form</title>
+    <style>
+      body > div#root {height: 100vh !important;} 
```

Comment:
```
      body > div#root {height: 100vh !important;} 
```

games with fire)

Reply:
I know)) It was a war, as you can tell from the amount of commits I have here 🤣